updatezinfo.py

```diff
@@ -7,6 +7,15 @@
 from six.moves.urllib import request
 from six.moves.urllib import error as urllib_error
 
+try:
+    import dateutil
+except ImportError:
+    print("dateutil not installed locally, adding src to Python path")
```

Comment:
Not crazy about this solution. I'd be more inclined to let it error out and expect anyone running `updatezinfo` to do so with `dateutil` installed.

Alternatively we could add a `tox` environment instead.

All that said, presumably the reason you did it this way is that it works for now without messing around with the CI or trying to debug a tox job and `updatezinfo` is going to go away before the next major release anyway, so maybe not the worst thing in the world.

Reply:
Yup, the alternative would be to install locally dateutil in the CI, but given that it uses tox afterwards I had mixed fillings about it. 

As said, given that this is going to go away as soon as we land your PR I was hoping to just remove this altogether soon.